catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -351,7 +368,10 @@
                             !org.abego.treelayout.*,
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            org.codice.ddf.catalog.ui.metacard.internal,
+                            org.codice.ddf.catalog.ui.metacard.impl
```

Comment:
✏️ Ideally the `impl` package would not be exported.

Reply:
The AttachmentInfoImpl POJO is used by ListApplication. But I could work around that, since it's just a POJO. I could put a simple impl as an inner class in ListApplication.

Reply:
What is the alternative?

Reply:
An option would be to put the bundles into a library jar to embed but you would probably need to pull the interface out into a new bundle for that to all work.

Another option is to have other bundles wanting these implementations to private package this package into their bundles. 

But what you have works.  It just tightly couples other bundles to this bundles implementation.  As it is, bundles importing these implementations will always have to recompile if this bundle changes in a binary incompatible way.  Embedding or private packaging will protect those bundles from changes in this bundle and gives the option to recompile or not depending if the consuming bundle cares about the changes.